### ISSUE TITLE
.sr-only class (clip property) fails Google Chrome Lighthouse audit when used on form labels

### ISSUE THREAD

**Issue by chrisblakley** [twbs/bootstrap#28795]
- Operating system and version: macOS 10.14.3
- Browser and version: Chrome 74 (and Canary 76)
- Reduced test case: https://gearside.com/labels.html
- Some additional notes here: https://github.com/chrisblakley/Nebula/issues/1900

I have been using the `.sr-only` class on search input fields on my sites and have just noticed that Google Chrome Lighthouse reports were saying that those inputs did not have an associated label. Although the `<label>` elements do exist, it seems that the `.sr-only` class causes Lighthouse to note them as failing.

I made that test page linked above to confirm which properties the Lighthouse audit did not like and have found that the `clip: rect(0,0,0,0)` parameter of the `.sr-only` class is what causes this problem.

You can see this by loading the above test URL and running an Accessibility audit in Lighthouse.

## Here are a few quick screenshots to summarize

HTML:
![Screen Shot 2019-05-20 at 12 21 28 AM](https://user-images.githubusercontent.com/6349678/57996447-84e7a480-7a95-11e9-9991-c6a62a405b0e.png)

Page:
![Screen Shot 2019-05-20 at 12 21 38 AM](https://user-images.githubusercontent.com/6349678/57996456-90d36680-7a95-11e9-9e2a-0f7fd25bef0c.png)

Lighthouse Report:
![Screen Shot 2019-05-20 at 12 23 01 AM](https://user-images.githubusercontent.com/6349678/57996457-929d2a00-7a95-11e9-99d5-ef3dfc562177.png)
Note that only the `.sr-only` fails in the report, but the `.sr-only-noclip` passes.

Obviously I'm sure that the `clip` property is needed for an edge-case (or even a common case that I'm not aware of), but I wanted to mention this for anyone who is trying to improve Lighthouse scores.

**Comment by patrickhlauke** [comment]
Interesting - this appears to be a false negative from the related aXe test.

**Comment by XhmikosR** [comment]
If we 100% sure it's a bug upstream, can someone report it there?

**Comment by patrickhlauke** [comment]
Was in the middle of doing that ;)

**Comment by XhmikosR** [comment]
Might be worth tracking it on Lighthouse too so that the update the lib when a fix is available :)

**Comment by XhmikosR** [comment]
Opened a Lighthouse issue https://github.com/GoogleChrome/lighthouse/issues/9045

Closing since it's not our fault.
